src/lnlq.jl

```diff
@@ -292,6 +338,41 @@ function lnlq!(solver :: LnlqSolver{T,S}, A, b :: AbstractVector{T};
     # Compute w̄ₖ₊₁
     @kaxpby!(m, -cₖ₊₁, u, sₖ₊₁, w̄)
 
+    if σₑₛₜ > 0 && !complex_error_bnd
+      if transfer_to_craig
+        disc_x = τtildeₖ^2 - τₖ₊₁^2
+        if disc_x < 0
+          complex_error_bnd = true
+        else
+          err_x = √disc_x
+        end
```

Comment:
```suggestion
disc_x < 0 ? complex_error_bnd = true : err_x = √disc_x
```

You can do the same modification for `disc_y`.